### PR TITLE
Updated styling Duplicate of PR#160

### PR DESCRIPTION
**Pull request by pjayathissa**
Note that this is not to latest Figma UI. TODO

This just fixes the layout, random shadows, padding, and learn more to bottom

Screenshot 2020-03-23 at 00 21 39

### CODE REVIEW

File: client/ionic/src/pages/About.tsx [hunk 1]
```diff
@@ -44,14 +44,13 @@ const About: React.FC = () => {
                         </IonCardHeader>
                         <IonCardContent className="tl">
                           {screen.bodyTexts &&
-                            screen.bodyTexts.map(txt => <p>{txt}</p>)}
+                            screen.bodyTexts.map((txt, key) => (
+                              <p key={key}>{txt}</p>
```
Comment (SamMousa) [inline client/ionic/src/pages/About.tsx]: What is the key used for in styling?

Comment (sstur) [reply client/ionic/src/pages/About.tsx]: The key is just a react thing, sort of like an index for array elements, it doesn't affect styling.

File: client/ionic/src/pages/Menu.tsx [hunk 2]
```diff
@@ -11,20 +11,20 @@ const Menu: React.FC = () => {
...
-        <IonItem href="/triage" color="primary" className="pb3 tc ph5">
+        <IonItem href="/triage" color="primary" className="pb3 tc ph4">
           <IonLabel>Check Your Health</IonLabel>
         </IonItem>
-        <IonItem href="/menu-distress" color="primary" className="pb3 tc ph5">
+        <IonItem href="/menu-distress" color="primary" className="pb3 tc ph4">
           <IonLabel>Feeling Distressed?</IonLabel>
         </IonItem>
-        <IonItem href="/local_maps" color="primary" className="pb3 tc ph5">
+        <IonItem href="/local_maps" color="primary" className="pb3 tc ph4">
```
Comment (sstur) [inline client/ionic/src/pages/Menu.tsx]: While we're here should we make this href kebab-case like the rest? (although I guess we're planning to remove local maps for initial release, so take this with a grain of salt)

File: client/ionic/src/pages/Triage.tsx [hunk 3]
```diff
@@ -14,6 +14,19 @@ class Triage extends React.Component<{}, TriageState> {
   constructor(args: Readonly<{}>) {
     super(args);
     const survey = new Survey.ReactSurveyModel(triage);
+
+    // Update CSS classes
+    survey.onUpdateQuestionCssClasses.add((survey, options) => {
+      const classes = options.cssClasses;
+      classes.mainRoot += ' f6';
+      classes.root = 'sq-root';
+      classes.title += '';
```
Comment (sstur) [inline client/ionic/src/pages/Triage.tsx]: Nit: Any reason for these empty string lines?

File: client/ionic/src/pages/About.tsx (hunk 1 above)
Comment (pjayathissa) [reply client/ionic/src/pages/About.tsx]: I was just sorting out the jsx error while I was at it. elements within a map require a key

File: client/ionic/src/pages/Triage.tsx (hunk 3 above)
Comment (pjayathissa) [reply client/ionic/src/pages/Triage.tsx]: Left this as an issue for now Styling the Triage Form #178

This is still a work in progress, I ran out of time

File: client/ionic/src/pages/Menu.tsx (hunk 2 above)
Comment (pjayathissa) [reply client/ionic/src/pages/Menu.tsx]: I guess that can be done in a future PR

**Pull request merged**